src/main/g8/src/main/scala/$package$/server/package.scala

```diff
@@ -0,0 +1,41 @@
+package $package$
+
+import $package$.api.Api
+$if(add_caliban_endpoint.truthy)$
+import com.example.api.GraphQLApi
+$endif$
+import $package$.config.ApiConfig
+import akka.actor.ActorSystem
+import akka.http.scaladsl.Http
+$if(add_caliban_endpoint.truthy)$
+import akka.http.scaladsl.server.RouteConcatenation._
+$endif$
+import zio._
+import zio.config.Config
+
+package object server {
+
+  type HttpServer = Has[HttpServer.Service]
+
+  object HttpServer {
```

Comment:
We're a little inconsistent in style here with other definitions we have. e.g. `Api` is defined in a separate file, not in the package object file. and package object file has only a type alias.

So for consistent style we should either extract HttpServer into a separate file, or merge other services with respective package object files.

Reply:
Good point. I'll fix that.